src/module.common.ts

```diff
@@ -0,0 +1,141 @@
+import { NgModule, ModuleWithProviders } from '@angular/core';
```

Comment:
It would be better if Material 2 wasn't dominating the imports :/ Will try to think of a better solution.